BUILD.bazel

```diff
@@ -50,3 +50,14 @@ cc_test(
     ],
 )
 
+cc_test(
+    name = "sphereCollision_test",
+    srcs = [
+        "tests/sphere_test.cpp",
+    ],
+    deps = [
+        ":utils",
```

Comment:
So going back to this as you have not fixed it yet: ideally this should depend on the libraries you use, so in this case just sphere library (if you had one) and vector/ray/

Reply:
ok.